Description: Disable webpki-roots based tests
Author: Jan Niehusmann <jan@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Last-Update: 2021-05-12

--- rust-rustls-native-certs-0.5.0.orig/tests/compare_mozilla.rs
+++ rust-rustls-native-certs-0.5.0/tests/compare_mozilla.rs
@@ -66,6 +66,7 @@ fn to_map<'a>(anchors: &'a [webpki::Trus
     r
 }
 
+/*
 #[test]
 fn test_does_not_have_many_roots_unknown_by_mozilla() {
     let native = rustls_native_certs::load_native_certs()
@@ -95,7 +96,9 @@ fn test_does_not_have_many_roots_unknown
     println!("{:?} anchors present in native set but not mozilla ({}%)", missing_in_moz_roots, diff * 100.);
     assert!(diff < threshold, "too many unknown roots");
 }
+*/
 
+/*
 #[test]
 fn test_contains_most_roots_known_by_mozilla() {
     let native = rustls_native_certs::load_native_certs()
@@ -127,6 +130,7 @@ fn test_contains_most_roots_known_by_moz
              missing_in_native_roots, diff * 100.);
     assert!(diff < threshold, "too many missing roots");
 }
+*/
 
 #[test]
 fn util_list_certs() {
